connaisseur/base/connaisseur-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
this file has to go in https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces